### PR TITLE
e2e improvements

### PR DESCRIPTION
**Pull request by vrutkovs**
This PR updates `hack/e2e.sh` script to setup latest Cincinnati version on a temporary Openshift cluster and runs e2e tests (currently just a simple curl)

https://issues.redhat.com/browse/OTA-103

### CODE REVIEW

File: hack/e2e.sh [hunk 1]
```diff
@@ -5,20 +5,52 @@
...
+  else
+    sleep ${DELAY}
+    ATTEMPTS=$((ATTEMPTS-1))
+  fi
+done
 
 # Check that policy engine returns channel data respond
-GRAPH_URL="http://${CINCINNATI_PE_BASE_URL:-cincinnati-policy-engine}:${CINCINNATI_PE_PORT:-8081}/v1/graph"
-curl --verbose --header 'Accept:application/json' "${GRAPH_URL}?channel=a"
+curl --header 'Accept:application/json' "${GRAPH_URL}"
```
Comment (steveej) [inline hack/e2e.sh:56]: Do we want to pipe it to `jq` so ensure it's valid JSON?

**Review by steveej — COMMENTED** [review]
This looks like a very good start, thanks! We can postpone the more elaborate final graph verification if you prefer that.

File: hack/e2e.sh (hunk 1 above)
Comment (vrutkovs) [reply hack/e2e.sh:56]: Test container doesn't have jq. I'd rather have a proper compiled test built here, using some kind of BDD testing - e.g. https://crates.io/crates/fluid or https://crates.io/crates/stainless2

**Pull request merged**